com.civicrm.hrjobroles/CRM/Hrjobroles/BAO/HrJobRoles.php

```diff
@@ -47,6 +47,20 @@ public static function getContactRoles($cuid)  {
   }
 
   /**
+   * Check Contact if exist   .
+   *
+   * @param String $searchValue
+   * @param String $searchField
+   * @return Integer ( Contact ID or 0 if not exist)
+   */
+  public static function checkContact($searchValue, $searchField) {
```

Comment:
it may happen while i refactor and doing changes to code so moving this up was unintentional  ... for the name , yes I agree with you and I will change it .